src/test/java/org/testeditor/fixture/web/CapabilityIntegrationtest.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2017 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+
+package org.testeditor.fixture.web;
+
+import java.io.File;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxProfile;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+public class CapabilityIntegrationtest {
+    
+    
+    @Test
+    public void readCapabilitySuccesful() throws Exception {
+        
+        // given
+        WebDriverFixture fixture = new WebDriverFixture();
+        fixture.startBrowser("firefox");
+        WebDriver driver = fixture.getDriver();
+        String proxyHttpExpected = "http://mysystem.proxy.server";
+        String proxySslExpected = "http://mysystem_ssl.proxy.server";
+        int proxySslPortExpected = 108;
+        int proxyTypeExpected = 5;
+        
+        // when
+        Capabilities cap = ((RemoteWebDriver) driver).getCapabilities();
```

Comment:
I don't see anything done here, only reading profile, right?
In the when section the code should be executed that is actually tested, right?